src/PhpSigep/Services/Real/ConsultaCep.php

```diff
@@ -35,12 +35,11 @@ public function execute($cep)
                 $consultaCepResposta->setBairro(SoapClientFactory::convertEncoding($r->return->bairro));
                 $consultaCepResposta->setCep($r->return->cep);
                 $consultaCepResposta->setCidade(SoapClientFactory::convertEncoding($r->return->cidade));
-                $consultaCepResposta->setComplemento1(SoapClientFactory::convertEncoding($r->return->complemento));
+                $consultaCepResposta->setComplemento1(SoapClientFactory::convertEncoding(isset($r->return->complemento) ? $r->return->complemento  : null));
                 $consultaCepResposta->setComplemento2(SoapClientFactory::convertEncoding($r->return->complemento2));
                 $consultaCepResposta->setEndereco(SoapClientFactory::convertEncoding($r->return->end));
-                $consultaCepResposta->setId($r->return->id);
-                $consultaCepResposta->setUf($r->return->uf);
-                $result->setResult($consultaCepResposta);
+                $consultaCepResposta->setId(isset($r->return->id) ? $r->return->id  : null);
+                $consultaCepResposta->setUf($r->return->uf);//
```

Comment:
@barreirinha por favor remova esse dois caracteres extras no final `//`
```suggestion
                $consultaCepResposta->setUf($r->return->uf);
```

Reply:
PRONTO!